spec/components/validators/integer_setting_validator_spec.rb

```diff
@@ -21,9 +21,21 @@
 
       it "returns true if value is a valid integer" do
         expect(validator.valid_value?(1)).to eq(true)
-        expect(validator.valid_value?(-1)).to eq(true)
         expect(validator.valid_value?('1')).to eq(true)
-        expect(validator.valid_value?('-1')).to eq(true)
+      end
+
+      it "defaults min to 0" do
```

Comment:
> Also, changing min to 5 will result in failing tests now.

:+1: This was what I was going for. The test wasn't actually testing that the default is `0` so the description felt odd to me.